source/precalculus/sage/common.sage

```diff
@@ -72,4 +72,18 @@ class TBILPrecal:
     def small_complex(real_part=range(-5,6), 
                       imaginary_part=[i for i in range(-5,6) if i != 0],length=1):
         '''Generates a list of unique 2-tuples of conjugate pairs of complex (not real) numbers of the form a+bi.'''
-        return sample([(a+b*I,a-b*I) for a in real_part for b in imaginary_part],length)
\ No newline at end of file
+        return sample([(a+b*I,a-b*I) for a in real_part for b in imaginary_part],length)
+
+    @staticmethod
+    def inequality_string(ineq):
+        '''Turns inequalities like <= into \leq for nicer printing'''
+        if ineq=="<=":
+            return r"\leq"
+        elif ineq==">=":
+            return r"\leq"
+        elif ineq=="<":
+            return r"\lt"
+        elif ineq==">":
+            return r"\gt"
+        else:
+            return "Not an inequality symbol"
```

Comment:
```suggestion
        raise ValueError("Not an inequality symbol")
```

Reply:
We should also contrast with `CheckIt.shuffled_inequality` used at https://github.com/TeamBasedInquiryLearning/library/blob/fa1e46b57cde6a08ce5735a42366b9e40c4dd314/source/precalculus/exercises/outcomes/EQ/EQ4/generator.sage#L29 see also https://github.com/StevenClontz/checkit/blob/911954f9b98df090d2ddb4dbe4f0e2a97bd1c96d/dashboard/checkit/wrapper/wrapper.sage#L35